src/main/java/randoop/main/GenTests.java

```diff
@@ -516,6 +522,60 @@ public boolean handle(String[] args) {
           codeWriter,
           GenInputsAbstract.regression_test_basename,
           "Regression");
+
+      HashSet<String> flakyTests = codeWriter.getFlakyTests();
+
+      if (flakyTests.size() > 0) {
+        HashMap<TypedOperation, Integer> testOccurrences = new HashMap<>();
+        HashMap<TypedOperation, Integer> flakyOccurrences = new HashMap<>();
+
+        // Tally occurrences of all methods for if-idf
+        // Each method is counted once if it appears in a sequence test.
+        tallyOperationsInSequences(testOccurrences, explorer.getRegressionSequences());
+
+        // TODO: cxing handle Error Test Sequence tallying.
+        // tallyOperationsInSequences(testOccurrences, explorer.getErrorTestSequences());
+
+        List<ExecutableSequence> regressionSequences = explorer.getRegressionSequences();
+        for (String flakyTestNum : flakyTests) {
+          int testNum = Integer.parseInt(flakyTestNum.substring(4)); // length of test
+          // Tests start at 001
+          ExecutableSequence flakyTest = regressionSequences.get(testNum - 1);
+          HashSet<TypedOperation> ops = new HashSet<>();
+          SimpleList<Statement> statements = flakyTest.sequence.statements;
+          for (int i = 0; i < statements.size(); i++) {
+            if (!statements.get(i).getOperation().isMethodCall()) continue;
+            ops.add(statements.get(i).getOperation());
+          }
+
+          for (TypedOperation to : ops) {
+            if (flakyOccurrences.containsKey(to)) {
+              flakyOccurrences.put(to, flakyOccurrences.get(to) + 1);
+            } else {
+              flakyOccurrences.put(to, 1);
+            }
+          }
+        }
+
+        Map<String, Double> methodFlakyPercentage = new HashMap<>();
+        List<Entry<String, Double>> sortedMethodsbyFlakiness;
+
+        for (TypedOperation to : testOccurrences.keySet()) {
+          double flakyMethodOccurrences =
+              flakyOccurrences.containsKey(to) ? flakyOccurrences.get(to) : 0;
+          methodFlakyPercentage.put(
+              to.toParsableString(), flakyMethodOccurrences / testOccurrences.get(to));
+        }
+
+        sortedMethodsbyFlakiness = new ArrayList<>(methodFlakyPercentage.entrySet());
+        sortedMethodsbyFlakiness.sort(reverseOrder(Entry.comparingByValue()));
+
+        System.out.println();
+        System.out.println("Percentage of method flakiness in tests.");
+        for (Entry<String, Double> e : sortedMethodsbyFlakiness) {
```

Comment:
Should this only output the first few methods?  Otherwise it could be a huge amount of output.
Also, I think it needs a clearer explanation than "Percentage of method flakiness".
For example, what should a user do with this information?